### ISSUE TITLE
Bullhorn Branded Career Portal

### ISSUE THREAD

**Issue by Cmclark21** [bullhorn/career-portal#346]
We have just gone into Bullhorn and configured our branded career portal to have an extra division within our company take only the IT jobs that are posted by our recruiters. Unfortunately, all jobs are posted to the main site. In our app.json file, we have added additionaljobcriteria, field, value, and sort fields, however, we cannot bypass the issue of all of our jobs posting to ONLY one site and not both. Do we need to wait for a recruiter to post a job or will this happen simultaneously with the addition of another division in the branded career portal?

**Comment by charlesabarnes** [comment]
The additional job criteria filters down which jobs appear on a portal.  Removing the filters or making the filters inclusive of more options will allow the jobs to appear.

Please let me know if you have any additional questions about htis

**Comment by Cmclark21** [comment]
Charles, 
Thank you, would this still be applicable where we have three branded portals? We have three websites, right now all of our jobs are going to our parent company's portal and each own, respective portal (on a website). We are figuring out what the best way to configure our filters and fields to ensure that jobs are only going to their appropriate website and not also going to the parent company site

**Comment by charlesabarnes** [comment]
Yes, this applies applies to as many career portals as you can host.   

For example, lets say you have 3 Portals for different purposes: Healthcare, ITstaffing, and Nursing.  In Bullhorn you can configure your filter field (such as customText2) to be a multi select field.  Then you select which division that you want to post to.   You can then add those separate divisions to the app.json file for all 3 career portal sites to ensure they are receiving only the correct jobs.

Please let me know if you have any more questions.

**Comment by Cmclark21** [comment]
Charles, 

Would our jobs continue to post to the correct WEBSITE (only have one career portal) by creating this multi-select field? We have one portal, three websites, and want jobs to funnel to only their respective website. Our issue is, we are not sure how to do this without having three individual career portals.

**Comment by charlesabarnes** [comment]
Hello,

The portal pulls from Bullhorn.  Bullhorn does not *push* to a portal.   So unless the portals are configured to pull different jobs, they will all be similar.  You can do this with the same portal, but it will require custom coding to make the links to the different portals differ with a URL parameter or something similar.

**Comment by Cmclark21** [comment]
Where would I add custom code so that the portal pulls those jobs and sends them to the correct website? Moreover, what portion of my career portal (one for three sites and different jobs) would need to be modified?

**Comment by charlesabarnes** [comment]
Assuming you link to each of your job boards separately, if you add a URL param that the portal can use to determine which jobs to pull, you will want to modify [search.service](https://github.com/bullhorn/career-portal/blob/master/src/app/services/search/search.service.ts), the app module file, and possibly [settings service](https://github.com/bullhorn/career-portal/blob/master/src/app/services/settings/settings.service.ts).  

So basically what you may want to do is have is: 
`recruitingwebsite.com/careers?site=uno` 
`recruitingwebsite.com/careers?site=dos`  

and have code to manage the different parameter values.

Please let me know if you have any additional questions.

**Comment by Cmclark21** [comment]
The path to set a URL param does not exist for me in Siteground (career-portal, src, app, services, search). I do not have a src folder that leads to the app folder. Is there anything we can do to get that folder and modify it to set additional URL parameters for our jobs?

**Comment by charlesabarnes** [comment]
Hello,

A good place to start will probably be taking a look at our [customizing page](https://bullhorn.github.io/career-portal/customizing.html).  this will require closing the career portal repository, making changes to the code in the repository, then transpile the portal, and upload it

**Comment by SelectPharma** [comment]
Hi I am also trying to send different jobs to two different websites.  Please can you help me with this?  A very basic overview would be appreciated as I have limited coding experience.  Thank you!
